server/routes/testutils/appSetup.ts

```diff
@@ -32,6 +34,18 @@ class MockUserService extends UserService {
   }
 }
 
+class MockCourtRegisterService extends CourtRegisterService {
```

Comment:
extract to different file

Reply:
one of the nice things about using classes is that you can mock them quite nicely using jest:

https://github.com/ministryofjustice/hmpps-book-video-link/blob/main/backend/routes/amendBooking/changeDateAndTimeController.test.ts#L8-L13

We don't have a great way of injecting this in use of force